themes/digital.gov/layouts/partials/core/page-data.html

```diff
@@ -1,17 +1,14 @@
-{{/* Partials are really only included by page type. So at the moment, the only way to customize this component is by passing in new values through the partial and checking for them in this template:
+<!-- prettier-ignore-start -->
+{{/*
+  
+    The "edit" & "last updated" on section at the bottom of content pages.
+    See https://digital.gov/event/2023/08/17/uswds-monthly-call-august-2023/#page-data
 
-  Here are values that are passed in through the partial:
-
-  - `.email` — for specifying a different primary email address to be used in feedback emails
-  - `.cc` — for specifying an additional email address to be added to the feedback emails
-  - `.page_type` — for specifying the page type (not being used at the moment)
-  - `.branch` — for specifying which branch the build is on. This helps the edit button to point to the right branch when viewing a preview page.
+    @example Edit Last updated on Jun 13, 2023 at 5:45 p.m., ET
 */}}
-
-{{- $email_copy := (print "We value your feedback and open questions. Thank you for helping to make our pages better.") -}}
-
-
-<div id="page-data" class="branch-{{- .branch -}}" name="page-data">
-  <!-- Edit file on GitHub -->
-  <div class="edit-file hidden"></div>
+<!-- prettier-ignore-end -->
```

Comment:
A few notes:
1. Indents look one level too far.
2. Updating to latest version of `prettier` & `prettier-plugin-go-template` has support for multiline comments, so you don't need to ignore. In general we should avoid ignoring as much as possible ― although I totally get why it's used here.

Reply:
Extra indents are fixed, good to know about the package updates, added to the to do list 👍🏻